lib/site_template/Gemfile.erb

```diff
@@ -0,0 +1,24 @@
+source "https://rubygems.org"
+ruby RUBY_VERSION
+
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "<%= Jekyll::VERSION %>"
```

Comment:
By that I mean "no" on the ">=", it is not our job to break a users site, "~>" will give them the updates they need should there be a security issue or a major bug in that version.  ">=" will give them 4.0 before they are even ready and break everything, this lands on us and we get the blame.